charts/consul/test/unit/gateway-resources-job.bats

```diff
@@ -96,16 +95,13 @@ target=templates/gateway-resources-job.yaml
   local actual=$(echo "$spec" | jq '.[13]')
   [ "${actual}" = "\"foo: bar\"" ]
 
-  local actual=$(echo "$spec" | jq '.[14] | ."-tolerations=- key"')
-  [ "${actual}" = "\"bar\"" ]
-
-  local actual=$(echo "$spec" | jq '.[15]')
+  local actual=$(echo "$spec" | jq '.[14]')
```

Comment:
really minor, but since we're here would we want to modify these so we just check for the value in the `spec` array so that we're not tied to the array indices in the future?